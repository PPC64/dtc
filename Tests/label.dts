/dts-v1/;
/ {
	hdmi_mux_2: endpoint {
		remote-endpoint = "foo";
	};
	hdmi: hdmi@0120000 {
		#address-cells = <1>;
		#size-cells = <0>;
	}


};

&hdmi {
	remote-endpoint = <&hdmi_mux_2>;
};
