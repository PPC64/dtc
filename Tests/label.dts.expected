/dts-v1/;

/  {

	endpoint {

		remote-endpoint = "foo";
		phandle = <0x1>;
	};
	hdmi@0120000 {

		#address-cells = <0x1>;
		#size-cells = <0x0>;
		remote-endpoint = <0x1>;
	};
};
